db/migrate/20210104224344_migrate_statuses_for_tax_returns.rb

```diff
@@ -1,6 +1,6 @@
 class MigrateStatusesForTaxReturns < ActiveRecord::Migration[6.0]
   def up
-    MigrateStatuses.migrate_all
+    # MigrateStatuses.migrate_all
```

Comment:
this is good but maybe delete it better

`# This migration used to call MigrateStatuses but that class has since been deleted.`